### PR TITLE
ahmed2022re

### PR DESCRIPTION
**Pull request by wi25hoy**
## Project Name

Write the `<ahmed2022re>` here, and as well as rename the PR name to reflect the `<ahmed2022>` for quick reviewing. 

## Compiled PDF

Add the link to the compiled pdf here. Example : [Compiled Pdf](2022/<ahmed2022re>/journal/article.pdf) 
not available due to build problem.

## Metadata Checklist

Ensure that you have added the following metadata information (put an x to the checkbox items) in `<paper_citekey>/journal/metadata.yaml`.

- [x] `title` field exists and it starts with `[Re] `.
- [x] `authors` contain author information with affiliations.
- [x] [ORCID id](https://orcid.org/) is provided for authors.
- [x] One contact author is added using `*`.
- [x] Github `url` is provided in `code` section.
- [x] [SWH identifier](https://archive.softwareheritage.org/save/) is provided in `swh`.
- [x] `replication` section is populated with the necessary information about the paper you reproduced.
- [x] `abstract` field is populated with your paper abstract (including double quotes)
- [x] `domain` should contain "ML Reproducibility Challenge 2021"
- [x] `review` section contains the url to the OpenReview forum of your paper.
- [x] Fields `contributors`, `dates`, `article` and `journal` are kept blank, as those will be populated by the Area Chairs / ReScience editors.

## Build Errors
$ ./build_all.sh
Building ahmed2022re/
/c/Users/Waqas Ahmed/MLRC_2/MLRC/2021/ahmed2022re/journal
Reverting Windows console CPs to (in,out) = (65001,65001)
Win CP console initial and current in/out Win: (65001, 65001), (1252, 1252)
Coding system for system and terminal: 'CP1252'
Rc files read:
NONE
Latexmk: This is Latexmk, John Collins, 17 Mar. 2022. Version 4.77, version: 4.77.
No specific requests made, so using default for latexmk.
Latexmk: Doing main (small) clean up for 'article.tex'
Latexmk: Doing extra clean up (of final output files) for 'article.tex'
Latexmk: Doing main (small) clean up for 'content.tex'
Latexmk: Doing extra clean up (of final output files) for 'content.tex'
Latexmk: Doing main (small) clean up for 'header.tex'
Latexmk: Doing extra clean up (of final output files) for 'header.tex'
rm: cannot remove 'metadata.tex': No such file or directory
C:\Users\Waqas Ahmed\AppData\Local\Microsoft\WindowsApps\python3.exe: can't open file 'C:\Users\Waqas': [Errno 2] No such file or directory
make: *** [Makefile:18: metadata.tex] Error 2
Build complete
rm: cannot remove 'article.aux': No such file or directory
rm: cannot remove 'article.bbl': No such file or directory
rm: cannot remove 'article.blg': No such file or directory
rm: cannot remove 'article.fdb_latexmk': No such file or directory
rm: cannot remove 'article.fls': No such file or directory
rm: cannot remove 'article.out': No such file or directory
rm: cannot remove 'article.log': No such file or directory
rm: cannot remove 'article.run.xml': No such file or directory
rm: cannot remove 'article.bcf': No such file or directory
Building editorial/
/c/Users/Waqas Ahmed/MLRC_2/MLRC/2021/editorial/journal
Makefile:1: *** missing separator. Stop.
rm: cannot remove 'metadata.tex': No such file or directory
Makefile:1: *** missing separator. Stop.
Build complete
rm: cannot remove 'article.aux': No such file or directory
rm: cannot remove 'article.bbl': No such file or directory
rm: cannot remove 'article.blg': No such file or directory
rm: cannot remove 'article.fdb_latexmk': No such file or directory
rm: cannot remove 'article.fls': No such file or directory
rm: cannot remove 'article.out': No such file or directory
rm: cannot remove 'article.log': No such file or directory
rm: cannot remove 'article.run.xml': No such file or directory
rm: cannot remove 'article.bcf': No such file or directory

If your build had any errors locally, please add it here. You can upload your build.log in a [Pastebin](https://pastebin.com/) link and put it here.

./yaml-to-latex.py -i metadata.yaml -o metadata.tex

### CODE REVIEW

**Comment by koustuvsinha** [comment]
Hi @wi25hoy , your PR still modifies files from the `2020` folder. Please review [git fundamentals](https://medium.com/programming-club-iit-kanpur/a-guide-to-git-the-fundamentals-2d3db6b4df53) and fix the commits of your PR such that any change for your files should only be contained in your folder.

File: 2021/ahmed2022re/openreview/packages.tex [hunk 1]
```diff
@@ -0,0 +1,43 @@
+% Import packages here
+\PassOptionsToPackage{numbers, compress}{natbib}
```
Comment (koustuvsinha) [inline 2021/ahmed2022re/openreview/packages.tex:2]: No need to import natbib package, as it is handled by ReScience.cls.

File: 2021/ahmed2022re/openreview/packages.tex [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+% ready for submission
+% \usepackage{neurips_2019}
+
+% to compile a preprint version, e.g., for submission to arXiv, add add the
+% [preprint] option:
+%     \usepackage[preprint]{neurips_2019}
+
+% to compile a camera-ready version, add the [final] option, e.g.:
+\usepackage[final]{neurips_2019}
```
Comment (koustuvsinha) [inline 2021/ahmed2022re/openreview/packages.tex:11]: Please remove this line, as it conflicts with ReScience.cls package.

**Comment by wi25hoy** [comment]
since i am trying to revert my commit that modifies the file from folder 2020, it is showing me the path error. if this path is not correct then what should be the path i tried /MLRC only but its also not working. keeping in the fact that these deletion was done manually from git directly. 
Should i fork and clone the main ReScience/MLRC again and then try to add files and changes???


Waqas Ahmed@DESKTOP-4APMTJQ MINGW64 ~/MLRC_2/MLRC/2020 (main)
$ git revert 915c8b653665ee38e7500584f7ceead287750adb
error: invalid path '2020/sundar2021reproducibility/journal/Icon?'
fatal: revert failed

**Pull request closed without merging**